src/Shared/Process/ProcessEx.cs

```diff
@@ -47,7 +48,6 @@ public ProcessEx(ITestOutputHelper output, Process proc, TimeSpan timeout)
             proc.BeginOutputReadLine();
             proc.BeginErrorReadLine();
 
-            _exited = new TaskCompletionSource<int>(TaskCreationOptions.RunContinuationsAsynchronously);
```

Comment:
Technically if the process exited before this line then there would be a race and the TCS could be null which would hang anyone waiting on it.